src/hooks/v1/transactor/SafeTransferFromTx.ts

```diff
@@ -13,6 +13,24 @@ export function useSafeTransferFromTx(): TransactorInstance<{
 
   return ({ newOwnerAddress }, txOpts) => {
     if (!transactor || !projectId || !contracts?.Projects) {
+      const missingParam = !transactor
```

Comment:
This check should have been here too, just added it to this PR since it was a small change.

Reply:
Nice, looks great